pkg/resource/deploy/source_eval.go

```diff
@@ -70,16 +72,21 @@ type EvalRunInfo struct {
 }
 
 // NewEvalSource returns a planning source that fetches resources by evaluating a package with a set of args and
-// a confgiuration map.  This evaluation is performed using the given plugin context and may optionally use the
+// a configuration map.  This evaluation is performed using the given plugin context and may optionally use the
 // given plugin host (or the default, if this is nil).  Note that closing the eval source also closes the host.
-func NewEvalSource(plugctx *plugin.Context, runinfo *EvalRunInfo,
-	defaultProviderInfo map[tokens.Package]workspace.PluginSpec, dryRun bool,
+func NewEvalSource(
+	plugctx *plugin.Context,
+	runinfo *EvalRunInfo,
+	defaultProviderInfo map[tokens.Package]workspace.PluginSpec,
+	dryRun bool,
+	loader schema.ReferenceLoader,
 ) Source {
```

Comment:
contract.Requiref that the loader is not nil here

Reply:
Done ✅